From: Carsten Schoenert <c.schoenert@t-online.de>
Date: Sat, 29 Mar 2014 16:04:44 -0400
Subject: libnss: tuning mozilla/security/nss/*/*.def files

Related on a version dependency for libnss3-dev that isn't resolvable in the
current Wheezy version 7.3 we need to use the libnss inside the source. For
some reason the parsing and preparation of the various '*.def' files isn't
working. To get a proper build just preparing the files by hand to make the
linker happy.

This is just a hack, it fixes the problem but doesn't solve the real
problem.
---
 mozilla/nsprpub/lib/ds/plds.def                    |  64 ++-
 mozilla/nsprpub/lib/libc/src/plc.def               |  58 +--
 mozilla/nsprpub/pr/src/nspr.def                    | 118 +++--
 mozilla/nsprpub/pr/tests/dll/my.def                |  44 +-
 .../security/nss/cmd/libpkix/testutil/pkixutil.def |  46 +-
 mozilla/security/nss/lib/ckfw/builtins/nssckbi.def |  46 +-
 mozilla/security/nss/lib/ckfw/capi/nsscapi.def     |  46 +-
 mozilla/security/nss/lib/ckfw/nssmkey/nssmkey.def  |  46 +-
 mozilla/security/nss/lib/freebl/freebl.def         |  46 +-
 mozilla/security/nss/lib/freebl/freebl_hash.def    |  56 +--
 mozilla/security/nss/lib/nss/nss.def               | 558 ++++++++++-----------
 mozilla/security/nss/lib/nss/pkixpriv.def          |  18 +-
 mozilla/security/nss/lib/smime/smime.def           | 226 ++++-----
 .../security/nss/lib/softoken/legacydb/nssdbm.def  |  44 +-
 mozilla/security/nss/lib/softoken/softokn.def      |  46 +-
 mozilla/security/nss/lib/sqlite/sqlite.def         |  46 +-
 mozilla/security/nss/lib/ssl/ssl.def               | 170 +++----
 mozilla/security/nss/lib/util/nssutil.def          | 168 +++----
 18 files changed, 889 insertions(+), 957 deletions(-)

diff --git a/mozilla/nsprpub/lib/ds/plds.def b/mozilla/nsprpub/lib/ds/plds.def
index cc54a4d..9bba7f1 100644
--- a/mozilla/nsprpub/lib/ds/plds.def
+++ b/mozilla/nsprpub/lib/ds/plds.def
@@ -1,24 +1,20 @@
-;+#
-;+# This Source Code Form is subject to the terms of the Mozilla Public
-;+# License, v. 2.0. If a copy of the MPL was not distributed with this
-;+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
-;+#
-;+# OK, this file is meant to support SUN, LINUX, AIX, OS/2 and WINDOWS
-;+#   1. For all unix platforms, the string ";-"  means "remove this line"
-;+#   2. For all unix platforms, the string " DATA " will be removed from any 
-;+#     line on which it occurs.
-;+#   3. Lines containing ";+" will have ";+" removed on SUN and LINUX.
-;+#      On AIX, lines containing ";+" will be removed.
-;+#   4. For all unix platforms, the string ";;" will thave the ";;" removed.
-;+#   5. For all unix platforms, after the above processing has taken place,
-;+#    all characters after the first ";" on the line will be removed.
-;+#    And for AIX, the first ";" will also be removed.
-;+#  This file is passed directly to windows. Since ';' is a comment, all UNIX
-;+#   directives are hidden behind ";", ";+", and ";-"
-;+NSPR_4.0 {
-;+    global:
-LIBRARY plds4 ;-
-EXPORTS ;-
+#
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+#
+# OK, this file is meant to support SUN, LINUX, AIX, OS/2 and WINDOWS
+#   2. For all unix platforms, the string "" will be removed from any 
+#     line on which it occurs.
+#   3. Lines containing ";
+#      On AIX, lines containing ";
+#   4. For all unix platforms, the string "" will thave the ";
+#   5. For all unix platforms, after the above processing has taken place,
+#    all characters after the first ";
+#    And for AIX, the first ";
+#  This file is passed directly to windows. Since ';
+NSPR_4.0 {
+    global:
 PL_ArenaAllocate;
 PL_ArenaFinish;
 PL_ArenaGrow;
@@ -41,20 +37,20 @@ PL_HashTableRemove;
 PL_InitArenaPool;
 PL_NewHashTable;
 libVersionPoint;
-;+    local: *;
-;+};
-;+
-;+NSPR_4.1 {
-;+    global:
+    local: *;
+};
+
+NSPR_4.1 {
+    global:
 PL_HashTableLookupConst;
 PL_HashTableRawLookupConst;
-;+} NSPR_4.0;
-;+
-;+NSPR_4.8.5 {
-;+    global:
+} NSPR_4.0;
+
+NSPR_4.8.5 {
+    global:
 PL_ClearArenaPool;
-;+} NSPR_4.1;
-;+NSPR_4.9.6 {
-;+    global:
+} NSPR_4.1;
+NSPR_4.9.6 {
+    global:
 PL_SizeOfArenaPoolExcludingPool;
-;+} NSPR_4.8.5;
+} NSPR_4.8.5;
diff --git a/mozilla/nsprpub/lib/libc/src/plc.def b/mozilla/nsprpub/lib/libc/src/plc.def
index ac8aa7b..c2caefb 100644
--- a/mozilla/nsprpub/lib/libc/src/plc.def
+++ b/mozilla/nsprpub/lib/libc/src/plc.def
@@ -1,24 +1,20 @@
-;+#
-;+# This Source Code Form is subject to the terms of the Mozilla Public
-;+# License, v. 2.0. If a copy of the MPL was not distributed with this
-;+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
-;+#
-;+# OK, this file is meant to support SUN, LINUX, AIX, OS/2 and WINDOWS
-;+#   1. For all unix platforms, the string ";-"  means "remove this line"
-;+#   2. For all unix platforms, the string " DATA " will be removed from any 
-;+#     line on which it occurs.
-;+#   3. Lines containing ";+" will have ";+" removed on SUN and LINUX.
-;+#      On AIX, lines containing ";+" will be removed.
-;+#   4. For all unix platforms, the string ";;" will thave the ";;" removed.
-;+#   5. For all unix platforms, after the above processing has taken place,
-;+#    all characters after the first ";" on the line will be removed.
-;+#    And for AIX, the first ";" will also be removed.
-;+#  This file is passed directly to windows. Since ';' is a comment, all UNIX
-;+#   directives are hidden behind ";", ";+", and ";-"
-;+NSPR_4.0 {
-;+    global:
-LIBRARY plc4 ;-
-EXPORTS ;-
+#
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+#
+# OK, this file is meant to support SUN, LINUX, AIX, OS/2 and WINDOWS
+#   2. For all unix platforms, the string "" will be removed from any 
+#     line on which it occurs.
+#   3. Lines containing ";
+#      On AIX, lines containing ";
+#   4. For all unix platforms, the string "" will thave the ";
+#   5. For all unix platforms, after the above processing has taken place,
+#    all characters after the first ";
+#    And for AIX, the first ";
+#  This file is passed directly to windows. Since ';
+NSPR_4.0 {
+    global:
 PL_Base64Decode;
 PL_Base64Encode;
 PL_CreateOptState;
@@ -58,15 +54,15 @@ PL_strrchr;
 PL_strrstr;
 PL_strstr;
 libVersionPoint;
-;+    local: *;
-;+};
-;+
-;+NSPR_4.2 {
-;+    global:
+    local: *;
+};
+
+NSPR_4.2 {
+    global:
 PL_strtok_r;
-;+} NSPR_4.0;
-;+
-;+NSPR_4.7 {
-;+    global:
+} NSPR_4.0;
+
+NSPR_4.7 {
+    global:
 PL_CreateLongOptState;
-;+} NSPR_4.2;
+} NSPR_4.2;
diff --git a/mozilla/nsprpub/pr/src/nspr.def b/mozilla/nsprpub/pr/src/nspr.def
index 7777a3d..1b07959 100644
--- a/mozilla/nsprpub/pr/src/nspr.def
+++ b/mozilla/nsprpub/pr/src/nspr.def
@@ -1,25 +1,21 @@
-;+#
-;+# This Source Code Form is subject to the terms of the Mozilla Public
-;+# License, v. 2.0. If a copy of the MPL was not distributed with this
-;+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
-;+#
-;+# OK, this file is meant to support SUN, LINUX, AIX, OS/2 and WINDOWS
-;+#   1. For all unix platforms, the string ";-"  means "remove this line"
-;+#   2. For all unix platforms, the string " DATA " will be removed from any 
-;+#     line on which it occurs.
-;+#   3. Lines containing ";+" will have ";+" removed on SUN and LINUX.
-;+#      On AIX, lines containing ";+" will be removed.
-;+#   4. For all unix platforms, the string ";;" will thave the ";;" removed.
-;+#   5. For all unix platforms, after the above processing has taken place,
-;+#    all characters after the first ";" on the line will be removed.
-;+#    And for AIX, the first ";" will also be removed.
-;+#  This file is passed directly to windows. Since ';' is a comment, all UNIX
-;+#   directives are hidden behind ";", ";+", and ";-"
-;+#
-;+NSPR_4.0 {
-;+	global:
-LIBRARY nspr4 ;-
-EXPORTS ;-
+#
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+#
+# OK, this file is meant to support SUN, LINUX, AIX, OS/2 and WINDOWS
+#   2. For all unix platforms, the string "" will be removed from any 
+#     line on which it occurs.
+#   3. Lines containing ";
+#      On AIX, lines containing ";
+#   4. For all unix platforms, the string "" will thave the ";
+#   5. For all unix platforms, after the above processing has taken place,
+#    all characters after the first ";
+#    And for AIX, the first ";
+#  This file is passed directly to windows. Since ';
+#
+NSPR_4.0 {
+	global:
 		LL_MaxInt;
 		LL_MinInt;
 		LL_Zero;
@@ -384,19 +380,19 @@ EXPORTS ;-
 		PRP_NewNakedCondVar;
 		PRP_TryLock;
 		libVersionPoint;
-;+	local: *;
-;+};
-;+
-;+NSPRprivate {
-;+	global:
+	local: *;
+};
+
+NSPRprivate {
+	global:
 		GetExecutionEnvironment;
 		PT_FPrintStats;
 		SetExecutionEnvironment;
-;+	local: *;
-;+};
-;+
-;+NSPR_4.1 {
-;+	global:
+	local: *;
+};
+
+NSPR_4.1 {
+	global:
 		PR_ConnectContinue;
 		PR_CreateIOLayer;
 		PR_EmulateAcceptRead;
@@ -407,47 +403,47 @@ EXPORTS ;-
 		PR_GetNumberOfProcessors;
 		PR_ImportPipe;
 		PR_SetEnv;
-;+} NSPR_4.0;
-;+
-;+NSPR_4.3 {
-;+	global:
+} NSPR_4.0;
+
+NSPR_4.3 {
+	global:
 		LL_MaxUint;
 		PR_CallOnceWithArg;
 		PR_GetLibraryFilePathname;
-;+} NSPR_4.1;
-;+
-;+NSPR_4.4 {
-;+	global:
+} NSPR_4.1;
+
+NSPR_4.4 {
+	global:
 		PR_GetPathSeparator;
-;+} NSPR_4.3;
-;+
-;+NSPR_4.5 {
-;+	global:
+} NSPR_4.3;
+
+NSPR_4.5 {
+	global:
 		PR_EnumerateAddrInfo;
 		PR_FreeAddrInfo;
 		PR_GetAddrInfoByName;
 		PR_GetCanonNameFromAddrInfo;
-;+} NSPR_4.4;
-;+
-;+NSPR_4.6 {
-;+	global:
+} NSPR_4.4;
+
+NSPR_4.6 {
+	global:
 		PR_GetPhysicalMemorySize;
-;+} NSPR_4.5;
-;+NSPR_4.7 {
-;+	global:
+} NSPR_4.5;
+NSPR_4.7 {
+	global:
 		PR_ParseTimeStringToExplodedTime;
-;+} NSPR_4.6;
-;+NSPR_4.8 {
-;+	global:
+} NSPR_4.6;
+NSPR_4.8 {
+	global:
 		PR_AssertCurrentThreadOwnsLock;
 		PR_AssertCurrentThreadInMonitor;
-;+} NSPR_4.7;
-;+NSPR_4.8.9 {
-;+      global:
+} NSPR_4.7;
+NSPR_4.8.9 {
+      global:
                 PR_GetVersion;
-;+} NSPR_4.8;
-;+NSPR_4.9.2 {
-;+      global:
+} NSPR_4.8;
+NSPR_4.9.2 {
+      global:
 		PR_GetThreadName;
 		PR_SetCurrentThreadName;
-;+} NSPR_4.8.9;
+} NSPR_4.8.9;
diff --git a/mozilla/nsprpub/pr/tests/dll/my.def b/mozilla/nsprpub/pr/tests/dll/my.def
index 75b71f5..2fd9d7e 100644
--- a/mozilla/nsprpub/pr/tests/dll/my.def
+++ b/mozilla/nsprpub/pr/tests/dll/my.def
@@ -1,26 +1,22 @@
-;+#
-;+# This Source Code Form is subject to the terms of the Mozilla Public
-;+# License, v. 2.0. If a copy of the MPL was not distributed with this
-;+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
-;+#
-;+# OK, this file is meant to support SUN, LINUX, AIX, OS/2 and WINDOWS
-;+#   1. For all unix platforms, the string ";-"  means "remove this line"
-;+#   2. For all unix platforms, the string " DATA " will be removed from any 
-;+#     line on which it occurs.
-;+#   3. Lines containing ";+" will have ";+" removed on SUN and LINUX.
-;+#      On AIX, lines containing ";+" will be removed.
-;+#   4. For all unix platforms, the string ";;" will thave the ";;" removed.
-;+#   5. For all unix platforms, after the above processing has taken place,
-;+#    all characters after the first ";" on the line will be removed.
-;+#    And for AIX, the first ";" will also be removed.
-;+#  This file is passed directly to windows. Since ';' is a comment, all UNIX
-;+#   directives are hidden behind ";", ";+", and ";-"
-;+#
-;+MY_1.0 {
-;+	global:
-LIBRARY my ;-
-EXPORTS ;-
+#
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+#
+# OK, this file is meant to support SUN, LINUX, AIX, OS/2 and WINDOWS
+#   2. For all unix platforms, the string "" will be removed from any 
+#     line on which it occurs.
+#   3. Lines containing ";
+#      On AIX, lines containing ";
+#   4. For all unix platforms, the string "" will thave the ";
+#   5. For all unix platforms, after the above processing has taken place,
+#    all characters after the first ";
+#    And for AIX, the first ";
+#  This file is passed directly to windows. Since ';
+#
+MY_1.0 {
+	global:
 		My_GetValue;
 		My_SetValue;
-;+	local: *;
-;+};
+	local: *;
+};
diff --git a/mozilla/security/nss/cmd/libpkix/testutil/pkixutil.def b/mozilla/security/nss/cmd/libpkix/testutil/pkixutil.def
index 407d40a..4247c95 100644
--- a/mozilla/security/nss/cmd/libpkix/testutil/pkixutil.def
+++ b/mozilla/security/nss/cmd/libpkix/testutil/pkixutil.def
@@ -1,25 +1,21 @@
-;+#
-;+# This Source Code Form is subject to the terms of the Mozilla Public
-;+# License, v. 2.0. If a copy of the MPL was not distributed with this
-;+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
-;+#
-;+# OK, this file is meant to support SUN, LINUX, AIX and WINDOWS
-;+#   1. For all unix platforms, the string ";-"  means "remove this line"
-;+#   2. For all unix platforms, the string " DATA " will be removed from any 
-;+#	line on which it occurs.
-;+#   3. Lines containing ";+" will have ";+" removed on SUN and LINUX.
-;+#      On AIX, lines containing ";+" will be removed.  
-;+#   4. For all unix platforms, the string ";;" will thave the ";;" removed.
-;+#   5. For all unix platforms, after the above processing has taken place,
-;+#    all characters after the first ";" on the line will be removed.  
-;+#    And for AIX, the first ";" will also be removed.
-;+#  This file is passed directly to windows. Since ';' is a comment, all UNIX
-;+#   directives are hidden behind ";", ";+", and ";-"
-;+
-;+NSS_3.12 {               # NSS 3.12 release
-;+    global:
-LIBRARY pkixutil ;-
-EXPORTS	;-
+#
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+#
+# OK, this file is meant to support SUN, LINUX, AIX and WINDOWS
+#   2. For all unix platforms, the string "" will be removed from any 
+#	line on which it occurs.
+#   3. Lines containing ";
+#      On AIX, lines containing ";
+#   4. For all unix platforms, the string "" will thave the ";
+#   5. For all unix platforms, after the above processing has taken place,
+#    all characters after the first ";
+#    And for AIX, the first ";
+#  This file is passed directly to windows. Since ';
+
+NSS_3.12 {               # NSS 3.12 release
+    global:
 createBuildResult;
 createCRL;
 createCert;
@@ -43,6 +39,6 @@ testToStringHelper;
 PKIX_Cert2ASCII;
 PKIX_Error2ASCII;
 PKIX_String2ASCII;
-;+    local:
-;+       *;
-;+};
+    local:
+       *;
+};
diff --git a/mozilla/security/nss/lib/ckfw/builtins/nssckbi.def b/mozilla/security/nss/lib/ckfw/builtins/nssckbi.def
index 8692d4c..e1b67dd 100644
--- a/mozilla/security/nss/lib/ckfw/builtins/nssckbi.def
+++ b/mozilla/security/nss/lib/ckfw/builtins/nssckbi.def
@@ -1,26 +1,22 @@
-;+#
-;+# This Source Code Form is subject to the terms of the Mozilla Public
-;+# License, v. 2.0. If a copy of the MPL was not distributed with this
-;+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
-;+#
-;+# OK, this file is meant to support SUN, LINUX, AIX and WINDOWS
-;+#   1. For all unix platforms, the string ";-"  means "remove this line"
-;+#   2. For all unix platforms, the string " DATA " will be removed from any 
-;+#     line on which it occurs.
-;+#   3. Lines containing ";+" will have ";+" removed on SUN and LINUX.
-;+#      On AIX, lines containing ";+" will be removed.
-;+#   4. For all unix platforms, the string ";;" will thave the ";;" removed.
-;+#   5. For all unix platforms, after the above processing has taken place,
-;+#    all characters after the first ";" on the line will be removed.
-;+#    And for AIX, the first ";" will also be removed.
-;+#  This file is passed directly to windows. Since ';' is a comment, all UNIX
-;+#   directives are hidden behind ";", ";+", and ";-"
-;+
-;+NSS_3.1 {       # NSS 3.1 release
-;+    global:
-LIBRARY nssckbi ;-
-EXPORTS ;-
+#
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+#
+# OK, this file is meant to support SUN, LINUX, AIX and WINDOWS
+#   2. For all unix platforms, the string "" will be removed from any 
+#     line on which it occurs.
+#   3. Lines containing ";
+#      On AIX, lines containing ";
+#   4. For all unix platforms, the string "" will thave the ";
+#   5. For all unix platforms, after the above processing has taken place,
+#    all characters after the first ";
+#    And for AIX, the first ";
+#  This file is passed directly to windows. Since ';
+
+NSS_3.1 {       # NSS 3.1 release
+    global:
 C_GetFunctionList;
-;+    local:
-;+*;
-;+};
+    local:
+*;
+};
diff --git a/mozilla/security/nss/lib/ckfw/capi/nsscapi.def b/mozilla/security/nss/lib/ckfw/capi/nsscapi.def
index d7e68c7..e1b67dd 100644
--- a/mozilla/security/nss/lib/ckfw/capi/nsscapi.def
+++ b/mozilla/security/nss/lib/ckfw/capi/nsscapi.def
@@ -1,26 +1,22 @@
-;+#
-;+# This Source Code Form is subject to the terms of the Mozilla Public
-;+# License, v. 2.0. If a copy of the MPL was not distributed with this
-;+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
-;+#
-;+# OK, this file is meant to support SUN, LINUX, AIX and WINDOWS
-;+#   1. For all unix platforms, the string ";-"  means "remove this line"
-;+#   2. For all unix platforms, the string " DATA " will be removed from any 
-;+#     line on which it occurs.
-;+#   3. Lines containing ";+" will have ";+" removed on SUN and LINUX.
-;+#      On AIX, lines containing ";+" will be removed.
-;+#   4. For all unix platforms, the string ";;" will thave the ";;" removed.
-;+#   5. For all unix platforms, after the above processing has taken place,
-;+#    all characters after the first ";" on the line will be removed.
-;+#    And for AIX, the first ";" will also be removed.
-;+#  This file is passed directly to windows. Since ';' is a comment, all UNIX
-;+#   directives are hidden behind ";", ";+", and ";-"
-;+
-;+NSS_3.1 {       # NSS 3.1 release
-;+    global:
-LIBRARY nsscapi ;-
-EXPORTS ;-
+#
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+#
+# OK, this file is meant to support SUN, LINUX, AIX and WINDOWS
+#   2. For all unix platforms, the string "" will be removed from any 
+#     line on which it occurs.
+#   3. Lines containing ";
+#      On AIX, lines containing ";
+#   4. For all unix platforms, the string "" will thave the ";
+#   5. For all unix platforms, after the above processing has taken place,
+#    all characters after the first ";
+#    And for AIX, the first ";
+#  This file is passed directly to windows. Since ';
+
+NSS_3.1 {       # NSS 3.1 release
+    global:
 C_GetFunctionList;
-;+    local:
-;+*;
-;+};
+    local:
+*;
+};
diff --git a/mozilla/security/nss/lib/ckfw/nssmkey/nssmkey.def b/mozilla/security/nss/lib/ckfw/nssmkey/nssmkey.def
index 45d307f..0fe7062 100644
--- a/mozilla/security/nss/lib/ckfw/nssmkey/nssmkey.def
+++ b/mozilla/security/nss/lib/ckfw/nssmkey/nssmkey.def
@@ -1,26 +1,22 @@
-;+#
-;+# This Source Code Form is subject to the terms of the Mozilla Public
-;+# License, v. 2.0. If a copy of the MPL was not distributed with this
-;+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
-;+#
-;+# OK, this file is meant to support SUN, LINUX, AIX and WINDOWS
-;+#   1. For all unix platforms, the string ";-"  means "remove this line"
-;+#   2. For all unix platforms, the string " DATA " will be removed from any 
-;+#     line on which it occurs.
-;+#   3. Lines containing ";+" will have ";+" removed on SUN and LINUX.
-;+#      On AIX, lines containing ";+" will be removed.
-;+#   4. For all unix platforms, the string ";;" will thave the ";;" removed.
-;+#   5. For all unix platforms, after the above processing has taken place,
-;+#    all characters after the first ";" on the line will be removed.
-;+#    And for AIX, the first ";" will also be removed.
-;+#  This file is passed directly to windows. Since ';' is a comment, all UNIX
-;+#   directives are hidden behind ";", ";+", and ";-"
-;+
-;+NSSMKEY_3.0 {       # First release of nssmkey
-;+    global:
-LIBRARY nssmkey ;-
-EXPORTS ;-
+#
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+#
+# OK, this file is meant to support SUN, LINUX, AIX and WINDOWS
+#   2. For all unix platforms, the string "" will be removed from any 
+#     line on which it occurs.
+#   3. Lines containing ";
+#      On AIX, lines containing ";
+#   4. For all unix platforms, the string "" will thave the ";
+#   5. For all unix platforms, after the above processing has taken place,
+#    all characters after the first ";
+#    And for AIX, the first ";
+#  This file is passed directly to windows. Since ';
+
+NSSMKEY_3.0 {       # First release of nssmkey
+    global:
 C_GetFunctionList;
-;+    local:
-;+*;
-;+};
+    local:
+*;
+};
diff --git a/mozilla/security/nss/lib/freebl/freebl.def b/mozilla/security/nss/lib/freebl/freebl.def
index 164c843..c382fda 100644
--- a/mozilla/security/nss/lib/freebl/freebl.def
+++ b/mozilla/security/nss/lib/freebl/freebl.def
@@ -1,26 +1,22 @@
-;+#
-;+# This Source Code Form is subject to the terms of the Mozilla Public
-;+# License, v. 2.0. If a copy of the MPL was not distributed with this
-;+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
-;+#
-;+# OK, this file is meant to support SUN, LINUX, AIX and WINDOWS
-;+#   1. For all unix platforms, the string ";-"  means "remove this line"
-;+#   2. For all unix platforms, the string " DATA " will be removed from any 
-;+#	line on which it occurs.
-;+#   3. Lines containing ";+" will have ";+" removed on SUN and LINUX.
-;+#      On AIX, lines containing ";+" will be removed.  
-;+#   4. For all unix platforms, the string ";;" will thave the ";;" removed.
-;+#   5. For all unix platforms, after the above processing has taken place,
-;+#    all characters after the first ";" on the line will be removed.  
-;+#    And for AIX, the first ";" will also be removed.
-;+#  This file is passed directly to windows. Since ';' is a comment, all UNIX
-;+#   directives are hidden behind ";", ";+", and ";-"
-;+
-;+NSSprivate_3.11 {               # NSS 3.11 release
-;+    global:
-LIBRARY freebl3 ;-
-EXPORTS	;-
+#
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+#
+# OK, this file is meant to support SUN, LINUX, AIX and WINDOWS
+#   2. For all unix platforms, the string "" will be removed from any 
+#	line on which it occurs.
+#   3. Lines containing ";
+#      On AIX, lines containing ";
+#   4. For all unix platforms, the string "" will thave the ";
+#   5. For all unix platforms, after the above processing has taken place,
+#    all characters after the first ";
+#    And for AIX, the first ";
+#  This file is passed directly to windows. Since ';
+
+NSSprivate_3.11 {               # NSS 3.11 release
+    global:
 FREEBL_GetVector;
-;+    local:
-;+       *;
-;+};
+    local:
+       *;
+};
diff --git a/mozilla/security/nss/lib/freebl/freebl_hash.def b/mozilla/security/nss/lib/freebl/freebl_hash.def
index 9fd2736..c80f5c9 100644
--- a/mozilla/security/nss/lib/freebl/freebl_hash.def
+++ b/mozilla/security/nss/lib/freebl/freebl_hash.def
@@ -1,31 +1,27 @@
-;+#
-;+# This Source Code Form is subject to the terms of the Mozilla Public
-;+# License, v. 2.0. If a copy of the MPL was not distributed with this
-;+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
-;+#
-;+# OK, this file is meant to support SUN, LINUX, AIX and WINDOWS
-;+#   1. For all unix platforms, the string ";-"  means "remove this line"
-;+#   2. For all unix platforms, the string " DATA " will be removed from any 
-;+#	line on which it occurs.
-;+#   3. Lines containing ";+" will have ";+" removed on SUN and LINUX.
-;+#      On AIX, lines containing ";+" will be removed.  
-;+#   4. For all unix platforms, the string ";;" will thave the ";;" removed.
-;+#   5. For all unix platforms, after the above processing has taken place,
-;+#    all characters after the first ";" on the line will be removed.  
-;+#    And for AIX, the first ";" will also be removed.
-;+#  This file is passed directly to windows. Since ';' is a comment, all UNIX
-;+#   directives are hidden behind ";", ";+", and ";-"
-;+
-;+NSSprivate_3.11 {               # NSS 3.11 release
-;+    global:
-LIBRARY freebl3 ;-
-EXPORTS	;-
+#
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+#
+# OK, this file is meant to support SUN, LINUX, AIX and WINDOWS
+#   2. For all unix platforms, the string "" will be removed from any 
+#	line on which it occurs.
+#   3. Lines containing ";
+#      On AIX, lines containing ";
+#   4. For all unix platforms, the string "" will thave the ";
+#   5. For all unix platforms, after the above processing has taken place,
+#    all characters after the first ";
+#    And for AIX, the first ";
+#  This file is passed directly to windows. Since ';
+
+NSSprivate_3.11 {               # NSS 3.11 release
+    global:
 FREEBL_GetVector;
-;+    local:
-;+       *;
-;+};
-;+NSSRAWHASH_3.12.3 {             # NSS 3.12.3 release
-;+    global:
+    local:
+       *;
+};
+NSSRAWHASH_3.12.3 {             # NSS 3.12.3 release
+    global:
 NSSLOW_Init;
 NSSLOW_Shutdown;
 NSSLOWHASH_Length;
@@ -34,6 +30,6 @@ NSSLOWHASH_Destroy;
 NSSLOWHASH_End;
 NSSLOWHASH_NewContext;
 NSSLOWHASH_Update;
-;+    local:
-;+       *;
-;+};
+    local:
+       *;
+};
diff --git a/mozilla/security/nss/lib/nss/nss.def b/mozilla/security/nss/lib/nss/nss.def
index 6f6b670..1507a30 100644
--- a/mozilla/security/nss/lib/nss/nss.def
+++ b/mozilla/security/nss/lib/nss/nss.def
@@ -1,24 +1,20 @@
-;+#
-;+# This Source Code Form is subject to the terms of the Mozilla Public
-;+# License, v. 2.0. If a copy of the MPL was not distributed with this
-;+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
-;+#
-;+# OK, this file is meant to support SUN, LINUX, AIX and WINDOWS
-;+#   1. For all unix platforms, the string ";-"  means "remove this line"
-;+#   2. For all unix platforms, the string " DATA " will be removed from any 
-;+#	line on which it occurs.
-;+#   3. Lines containing ";+" will have ";+" removed on SUN and LINUX.
-;+#      On AIX, lines containing ";+" will be removed.  
-;+#   4. For all unix platforms, the string ";;" will thave the ";;" removed.
-;+#   5. For all unix platforms, after the above processing has taken place,
-;+#    all characters after the first ";" on the line will be removed.  
-;+#    And for AIX, the first ";" will also be removed.
-;+#  This file is passed directly to windows. Since ';' is a comment, all UNIX
-;+#   directives are hidden behind ";", ";+", and ";-"
-;+NSS_3.2 {       # NSS 3.2 release
-;+    global:
-LIBRARY nss3	;-
-EXPORTS		;-
+#
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+#
+# OK, this file is meant to support SUN, LINUX, AIX and WINDOWS
+#   2. For all unix platforms, the string "" will be removed from any 
+#	line on which it occurs.
+#   3. Lines containing ";
+#      On AIX, lines containing ";
+#   4. For all unix platforms, the string "" will thave the ";
+#   5. For all unix platforms, after the above processing has taken place,
+#    all characters after the first ";
+#    And for AIX, the first ";
+#  This file is passed directly to windows. Since ';
+NSS_3.2 {       # NSS 3.2 release
+    global:
 ATOB_AsciiToData;
 BTOA_ConvertItemToAscii;
 BTOA_DataToAscii;
@@ -144,10 +140,10 @@ SEC_FindCrlByDERCert;
 SEC_FindCrlByName;
 SEC_LookupCrls;
 SEC_NewCrl;
-;+#
-;+# The following symbols are exported only to make libssl3.so work. 
-;+# These are still private!!!
-;+#
+#
+# The following symbols are exported only to make libssl3.so work. 
+# These are still private!!!
+#
 __CERT_NewTempCertificate;
 __PK11_CreateContextByRawKey;
 __PK11_GetKeyData;
@@ -236,10 +232,10 @@ VFY_CreateContext;
 VFY_DestroyContext;
 VFY_End;
 VFY_Update;
-;+#
-;+# The following symbols are exported only to make libsmime3.so work. 
-;+# These are still private!!!
-;+#
+#
+# The following symbols are exported only to make libsmime3.so work. 
+# These are still private!!!
+#
 __CERT_ClosePermCertDB;
 __CERT_DecodeDERCertificate;
 __CERT_TraversePermCertsForNickname;
@@ -337,37 +333,37 @@ SGN_DestroyDigestInfo;
 SGN_Digest;
 VFY_VerifyData;
 VFY_VerifyDigest;
-;+#
-;+# Data objects
-;+#
-;+# Don't export these DATA symbols on Windows because they don't work right.
-;+# Use the SEC_ASN1_GET / SEC_ASN1_SUB / SEC_ASN1_XTRN macros to access them.
-;;CERT_CrlTemplate DATA ;
-;;CERT_SignedDataTemplate DATA ;
-;;CERT_CertificateTemplate DATA ;
-;;CERT_CertificateRequestTemplate DATA ;
-;;CERT_IssuerAndSNTemplate DATA ;
-;;CERT_SetOfSignedCrlTemplate DATA ;
-;;SECKEY_DSAPublicKeyTemplate DATA ;
-;;SECKEY_EncryptedPrivateKeyInfoTemplate DATA ;
-;;SECKEY_PointerToEncryptedPrivateKeyInfoTemplate DATA ;
-;;SECKEY_PointerToPrivateKeyInfoTemplate DATA ;
-;;SECKEY_PrivateKeyInfoTemplate DATA ;
-;;SECKEY_RSAPublicKeyTemplate DATA ;
-;;SECOID_AlgorithmIDTemplate DATA ;
-;;SEC_AnyTemplate DATA ;
-;;SEC_BMPStringTemplate DATA ;
-;;SEC_BitStringTemplate DATA ;
-;;SEC_GeneralizedTimeTemplate DATA ;
-;;SEC_IA5StringTemplate DATA ;
-;;SEC_IntegerTemplate DATA ;
-;;SEC_ObjectIDTemplate DATA ;
-;;SEC_OctetStringTemplate DATA ;
-;;SEC_PointerToAnyTemplate DATA ;
-;;SEC_PointerToOctetStringTemplate DATA ;
-;;SEC_SetOfAnyTemplate DATA ;
-;;SEC_UTCTimeTemplate DATA ;
-;;sgn_DigestInfoTemplate DATA ;
+#
+# Data objects
+#
+# Don't export thesesymbols on Windows because they don't work right.
+# Use the SEC_ASN1_GET / SEC_ASN1_SUB / SEC_ASN1_XTRN macros to access them.
+CERT_CrlTemplate;
+CERT_SignedDataTemplate;
+CERT_CertificateTemplate;
+CERT_CertificateRequestTemplate;
+CERT_IssuerAndSNTemplate;
+CERT_SetOfSignedCrlTemplate;
+SECKEY_DSAPublicKeyTemplate;
+SECKEY_EncryptedPrivateKeyInfoTemplate;
+SECKEY_PointerToEncryptedPrivateKeyInfoTemplate;
+SECKEY_PointerToPrivateKeyInfoTemplate;
+SECKEY_PrivateKeyInfoTemplate;
+SECKEY_RSAPublicKeyTemplate;
+SECOID_AlgorithmIDTemplate;
+SEC_AnyTemplate;
+SEC_BMPStringTemplate;
+SEC_BitStringTemplate;
+SEC_GeneralizedTimeTemplate;
+SEC_IA5StringTemplate;
+SEC_IntegerTemplate;
+SEC_ObjectIDTemplate;
+SEC_OctetStringTemplate;
+SEC_PointerToAnyTemplate;
+SEC_PointerToOctetStringTemplate;
+SEC_SetOfAnyTemplate;
+SEC_UTCTimeTemplate;
+sgn_DigestInfoTemplate;
 NSS_Get_CERT_CrlTemplate;
 NSS_Get_CERT_SignedDataTemplate;
 NSS_Get_CERT_CertificateTemplate;
@@ -394,7 +390,7 @@ NSS_Get_SEC_PointerToOctetStringTemplate;
 NSS_Get_SEC_SetOfAnyTemplate;
 NSS_Get_SEC_UTCTimeTemplate;
 NSS_Get_sgn_DigestInfoTemplate;
-;+# commands
+# commands
 CERT_DecodeBasicConstraintValue;
 CERT_DecodeOidSequence;
 CERT_DecodeUserNotice;
@@ -413,34 +409,34 @@ NSSBase64Decoder_Update;
 NSSBase64Encoder_Create;
 NSSBase64Encoder_Destroy;
 NSSBase64Encoder_Update;
-;+#PK11_DoPassword;
-;+#PK11_FindKeyByKeyID;
+#PK11_DoPassword;
+#PK11_FindKeyByKeyID;
 PK11_InitPin;
 PK11_NeedUserInit;
-;+    local:
-;+       *;
-;+};
-;+NSS_3.2.1 {       # NSS 3.2.1 release
-;+    global:
+    local:
+       *;
+};
+NSS_3.2.1 {       # NSS 3.2.1 release
+    global:
 CERT_AddRDN;
 CERT_CreateRDN;
 CERT_CreateAVA;
 CERT_CreateName;
-;+    local:
-;+       *;
-;+};
-;+NSS_3.3 { 	# NSS 3.3. release
-;+    global:
+    local:
+       *;
+};
+NSS_3.3 { 	# NSS 3.3. release
+    global:
 CERT_CheckCertUsage;
 CERT_FindCertIssuer;
 PK11_GetModule;
 SECKEY_CreateDHPrivateKey;
 SECKEY_GetPublicKeyType;
 SECMOD_AddNewModule;
-;+#
-;+# The following symbols are exported only to make JSS work.
-;+# These are still private!!!
-;+#
+#
+# The following symbols are exported only to make JSS work.
+# These are still private!!!
+#
 CERT_DisableOCSPChecking;
 CERT_DisableOCSPDefaultResponder;
 CERT_EnableOCSPDefaultResponder;
@@ -516,22 +512,22 @@ SECKEY_ImportDERPublicKey;
 SECKEY_NewPrivateKeyList;
 SECKEY_RemovePrivateKeyListNode;
 VFY_EndWithSignature;
-;+    local:
-;+       *;
-;+};
-;+NSS_3.3.1 { 	# NSS 3.3.1 release
-;+    global:
-;+#
-;+# The following symbols are exported only to make libsmime3.so work. 
-;+# These are still private!!!
-;+#
+    local:
+       *;
+};
+NSS_3.3.1 { 	# NSS 3.3.1 release
+    global:
+#
+# The following symbols are exported only to make libsmime3.so work. 
+# These are still private!!!
+#
 PK11_CreatePBEParams;
 PK11_DestroyPBEParams;
-;+    local:
-;+       *;
-;+};
-;+NSS_3.4 { 	# NSS 3.4 release
-;+    global:
+    local:
+       *;
+};
+NSS_3.4 { 	# NSS 3.4 release
+    global:
 SECMOD_AddNewModuleEx;
 SECMOD_DeleteModule;
 SECMOD_FreeModuleSpecList;
@@ -540,15 +536,15 @@ SECMOD_LoadModule;
 SECMOD_LoadUserModule;
 SECMOD_UnloadUserModule;
 SECMOD_UpdateModule;
-;+# for PKCS #12
+# for PKCS #12
 PK11_RawPBEKeyGen;
-;+# for PSM
+# for PSM
 __CERT_AddTempCertToPerm;
 CERT_AddOKDomainName;
 CERT_CopyName;
 CERT_CreateSubjectCertList;
 CERT_DecodeAVAValue;
-;+#CERT_DecodeCertFromPackage;
+#CERT_DecodeCertFromPackage;
 CERT_DecodeGeneralName;
 CERT_DecodeTrustString;
 CERT_DerNameToAscii;
@@ -572,8 +568,8 @@ HASH_Destroy;
 HASH_End;
 HASH_ResultLen;
 HASH_Update;
-NSSBase64_DecodeBuffer;   # from Stan
-NSSBase64_EncodeItem;   # from Stan
+NSSBase64_DecodeBuffer;
+NSSBase64_EncodeItem;
 PK11_GetKeyGen;
 PK11_GetMinimumPwdLength;
 PK11_GetNextSafe;
@@ -593,19 +589,19 @@ SECMOD_FindModule;
 SECMOD_FindSlot;
 SECMOD_PubCipherFlagstoInternal;
 SECMOD_PubMechFlagstoInternal;
-;;CERT_NameTemplate DATA ;
-;;CERT_SubjectPublicKeyInfoTemplate DATA ;
-;;SEC_BooleanTemplate DATA ;
-;;SEC_NullTemplate DATA ;
-;;SEC_SignedCertificateTemplate DATA ;
-;;SEC_UTF8StringTemplate DATA ;
+CERT_NameTemplate;
+CERT_SubjectPublicKeyInfoTemplate;
+SEC_BooleanTemplate;
+SEC_NullTemplate;
+SEC_SignedCertificateTemplate;
+SEC_UTF8StringTemplate;
 NSS_Get_CERT_NameTemplate;
 NSS_Get_CERT_SubjectPublicKeyInfoTemplate;
 NSS_Get_SEC_BooleanTemplate;
 NSS_Get_SEC_NullTemplate;
 NSS_Get_SEC_SignedCertificateTemplate;
 NSS_Get_SEC_UTF8StringTemplate;
-;+# for JSS
+# for JSS
 PK11_DeleteTokenPrivateKey;
 PK11_DeleteTokenPublicKey;
 PK11_DeleteTokenSymKey;
@@ -627,13 +623,13 @@ PK11_SetPublicKeyNickname;
 PK11_SetSymKeyNickname;
 SECKEY_DecodeDERSubjectPublicKeyInfo;
 SECKEY_DestroyPublicKeyList;
-;+# for debugging
+# for debugging
 nss_DumpCertificateCacheInfo;
-;+    local:
-;+       *;
-;+};
-;+NSS_3.5 { 	# cert creation APIs used by certutil
-;+    global:
+    local:
+       *;
+};
+NSS_3.5 { 	# cert creation APIs used by certutil
+    global:
 CERT_AddExtension;
 CERT_CopyRDN;
 CERT_CreateCertificate;
@@ -649,11 +645,11 @@ DER_AsciiToTime;
 PK11_ImportCert;
 PORT_Strdup;
 SECMOD_CanDeleteInternalModule;
-;+    local:
-;+       *;
-;+};
-;+NSS_3.6 { 	# NSS 3.6 release
-;+    global:
+    local:
+       *;
+};
+NSS_3.6 { 	# NSS 3.6 release
+    global:
 CERT_AddOCSPAcceptableResponses;
 CERT_CompleteCRLDecodeEntries;
 CERT_CreateOCSPCertID;
@@ -683,11 +679,11 @@ PK11_SaveContextAlloc;
 PK11_TokenKeyGen;
 SEC_QuickDERDecodeItem;
 SECKEY_CopyPublicKey;
-;+    local:
-;+       *;
-;+};
-;+NSS_3.7 { 	# NSS 3.7 release
-;+    global:
+    local:
+       *;
+};
+NSS_3.7 { 	# NSS 3.7 release
+    global:
 CERT_CRLCacheRefreshIssuer;
 CERT_DestroyOCSPResponse;
 CERT_EncodeAltNameExtension;
@@ -698,17 +694,17 @@ CERT_GetNextEmailAddress;
 CERT_VerifySignedDataWithPublicKey;
 CERT_VerifySignedDataWithPublicKeyInfo;
 PK11_WaitForTokenEvent;
-;+    local:
-;+       *;
-;+};
-;+NSS_3.7.1 { 	# NSS 3.7.1 release
-;+    global:
+    local:
+       *;
+};
+NSS_3.7.1 { 	# NSS 3.7.1 release
+    global:
 PK11_TokenRefresh;
-;+    local:
-;+       *;
-;+};
-;+NSS_3.8 { 	# NSS 3.8 release
-;+    global:
+    local:
+       *;
+};
+NSS_3.8 { 	# NSS 3.8 release
+    global:
 CERT_IsRootDERCert;
 HASH_GetHashObjectByOidTag;
 HASH_GetHashTypeByOidTag;
@@ -721,14 +717,14 @@ PK11_UserDisableSlot;
 SECITEM_ItemsAreEqual;
 SECKEY_CreateECPrivateKey;
 SECKEY_PublicKeyStrengthInBits;
-;+    local:
-;+       *;
-;+};
-;+NSS_3.9 { 	# NSS 3.9 release
-;+    global:
+    local:
+       *;
+};
+NSS_3.9 { 	# NSS 3.9 release
+    global:
 CERT_DestroyOidSequence;
 CERT_GetOidString;
-;;CERT_TimeChoiceTemplate DATA ;
+CERT_TimeChoiceTemplate;
 DER_DecodeTimeChoice;
 DER_EncodeTimeChoice;
 DSAU_DecodeDerSigToLen;
@@ -748,11 +744,11 @@ SECMOD_GetDeadModuleList;
 SEC_ASN1DecoderAbort;
 SEC_ASN1EncoderAbort;
 SEC_DupCrl;
-;+    local:
-;+       *;
-;+};
-;+NSS_3.9.2 { 	# NSS 3.9.2 release
-;+    global:
+    local:
+       *;
+};
+NSS_3.9.2 { 	# NSS 3.9.2 release
+    global:
 NSS_IsInitialized;
 PK11_DestroyGenericObject;
 PK11_DestroyGenericObjects;
@@ -762,11 +758,11 @@ PK11_GetPrevGenericObject;
 PK11_LinkGenericObject;
 PK11_ReadRawAttribute;
 PK11_UnlinkGenericObject;
-;+    local:
-;+       *;
-;+};
-;+NSS_3.9.3 { 	# NSS 3.9.3 release
-;+    global:
+    local:
+       *;
+};
+NSS_3.9.3 { 	# NSS 3.9.3 release
+    global:
 PK11_GetCertFromPrivateKey;
 PK11_PrivDecryptPKCS1;
 PK11_PubEncryptPKCS1;
@@ -774,11 +770,11 @@ SECMOD_CancelWait;
 SECMOD_HasRemovableSlots;
 SECMOD_UpdateSlotList;
 SECMOD_WaitForAnyTokenEvent;
-;+    local:
-;+       *;
-;+};
-;+NSS_3.10 { 	# NSS 3.10 release
-;+    global:
+    local:
+       *;
+};
+NSS_3.10 { 	# NSS 3.10 release
+    global:
 CERT_CacheCRL;
 CERT_DecodeAltNameExtension;
 CERT_DecodeAuthInfoAccessExtension;
@@ -806,27 +802,27 @@ HASH_ResultLenContext;
 SEC_GetSignatureAlgorithmOidTag;
 SECKEY_CacheStaticFlags;
 SECOID_AddEntry;
-;+#
-;+# Data objects
-;+#
-;+# Don't export these DATA symbols on Windows because they don't work right.
-;+# Use the SEC_ASN1_GET / SEC_ASN1_SUB / SEC_ASN1_XTRN macros to access them.
-;;CERT_SequenceOfCertExtensionTemplate DATA ;
-;;CERT_SignedCrlTemplate DATA ;
+#
+# Data objects
+#
+# Don't export thesesymbols on Windows because they don't work right.
+# Use the SEC_ASN1_GET / SEC_ASN1_SUB / SEC_ASN1_XTRN macros to access them.
+CERT_SequenceOfCertExtensionTemplate;
+CERT_SignedCrlTemplate;
 NSS_Get_CERT_SequenceOfCertExtensionTemplate;
 NSS_Get_CERT_SignedCrlTemplate;
-;+    local:
-;+       *;
-;+};
-;+NSS_3.10.2 { 	# NSS 3.10.2 release
-;+    global:
+    local:
+       *;
+};
+NSS_3.10.2 { 	# NSS 3.10.2 release
+    global:
 PK11_TokenKeyGenWithFlags;
 PK11_GenerateKeyPairWithFlags;
-;+    local:
-;+       *;
-;+};
-;+NSS_3.11 { 	# NSS 3.11 release
-;+    global:
+    local:
+       *;
+};
+NSS_3.11 { 	# NSS 3.11 release
+    global:
 CERT_CompareValidityTimes;
 PK11_CopyTokenPrivKeyToSessionPrivKey;
 PK11_FreeSlotListElement;
@@ -837,26 +833,26 @@ PK11_SetSymKeyUserData;
 SECMOD_CloseUserDB;
 SECMOD_HasRootCerts;
 SECMOD_OpenUserDB;
-;+    local:
-;+       *;
-;+};
-;+NSS_3.11.1 {
-;+    global:
+    local:
+       *;
+};
+NSS_3.11.1 {
+    global:
 NSS_RegisterShutdown;
 NSS_UnregisterShutdown;
 SEC_ASN1EncodeUnsignedInteger;
 SEC_RegisterDefaultHttpClient;
-;+    local:
-;+       *;
-;+};
-;+NSS_3.11.2 {
-;+    global:
+    local:
+       *;
+};
+NSS_3.11.2 {
+    global:
 SECKEY_SignatureLen;
-;+    local:
-;+       *;
-;+};
-;+NSS_3.11.7 {
-;+    global:
+    local:
+       *;
+};
+NSS_3.11.7 {
+    global:
 CERT_SetOCSPFailureMode;
 CERT_OCSPCacheSettings;
 CERT_ClearOCSPCache;
@@ -864,17 +860,17 @@ DER_GeneralizedDayToAscii;
 DER_TimeChoiceDayToAscii;
 DER_TimeToGeneralizedTime;
 DER_TimeToGeneralizedTimeArena;
-;+    local:
-;+       *;
-;+};
-;+NSS_3.11.9 {
-;+    global:
+    local:
+       *;
+};
+NSS_3.11.9 {
+    global:
 PK11_UnconfigurePKCS11;
-;+    local:
-;+       *;
-;+};
-;+NSS_3.12 { 	# NSS 3.12 release
-;+    global:
+    local:
+       *;
+};
+NSS_3.12 { 	# NSS 3.12 release
+    global:
 CERT_CheckNameSpace;
 CERT_EncodeCertPoliciesExtension;
 CERT_EncodeInfoAccessExtension;
@@ -920,145 +916,145 @@ VFY_VerifyDataDirect;
 VFY_VerifyDataWithAlgorithmID;
 VFY_VerifyDigestDirect;
 VFY_VerifyDigestWithAlgorithmID;
-;+    local:
-;+       *;
-;+};
-;+NSS_3.12.1 { 	# NSS 3.12.1 release
-;+    global:
+    local:
+       *;
+};
+NSS_3.12.1 { 	# NSS 3.12.1 release
+    global:
 CERT_NameToAsciiInvertible;
 PK11_FindCertFromDERCertItem;
-;+    local:
-;+       *;
-;+};
-;+NSS_3.12.3 { 	# NSS 3.12.3 release
-;+    global:
+    local:
+       *;
+};
+NSS_3.12.3 { 	# NSS 3.12.3 release
+    global:
 CERT_CompareCerts;
 CERT_RegisterAlternateOCSPAIAInfoCallBack;
 PK11_GetSymKeyHandle;
-;+    local:
-;+       *;
-;+};
-;+NSS_3.12.4 { 	# NSS 3.12.4 release
-;+    global:
+    local:
+       *;
+};
+NSS_3.12.4 { 	# NSS 3.12.4 release
+    global:
 PK11_IsInternalKeySlot;
 SECMOD_OpenNewSlot;
-;+    local:
-;+       *;
-;+};
-;+NSS_3.12.5 { 	# NSS 3.12.5 release
-;+    global:
+    local:
+       *;
+};
+NSS_3.12.5 { 	# NSS 3.12.5 release
+    global:
 CERT_AddCertToListSorted;
 NSS_InitContext;
 NSS_ShutdownContext;
 SECMOD_GetDefaultModDBFlag;
 SECMOD_GetSkipFirstFlag;
-;+    local:
-;+       *;
-;+};
-;+NSS_3.12.6 { 	# NSS 3.12.6 release
-;+    global:
+    local:
+       *;
+};
+NSS_3.12.6 { 	# NSS 3.12.6 release
+    global:
 CERT_CacheOCSPResponseFromSideChannel;
 CERT_DistNamesFromCertList;
 CERT_DupDistNames;
-;+    local:
-;+       *;
-;+};
-;+NSS_3.12.7 { 	# NSS 3.12.7 release
-;+    global:
+    local:
+       *;
+};
+NSS_3.12.7 { 	# NSS 3.12.7 release
+    global:
 CERT_GetConstrainedCertificateNames;
-;+    local:
-;+       *;
-;+};
-;+NSS_3.12.9 {  # NSS 3.12.9 release
-;+    global:
+    local:
+       *;
+};
+NSS_3.12.9 {  # NSS 3.12.9 release
+    global:
 CERT_FindCertByNicknameOrEmailAddrForUsage;
 PK11_DeriveWithTemplate;
 PK11_FindCertsFromEmailAddress;
 PK11_KeyGenWithTemplate;
 SECMOD_RestartModules;
-;+    local:
-;+       *;
-;+};
-;+NSS_3.12.10 {  # NSS 3.12.10 release
-;+    global:
+    local:
+       *;
+};
+NSS_3.12.10 {  # NSS 3.12.10 release
+    global:
 CERT_AllocCERTRevocationFlags;
 CERT_DestroyCERTRevocationFlags;
-;+    local:
-;+       *;
-;+};
-;+NSS_3.13 { 	# NSS 3.13 release
-;+    global:
-;;SECKEY_RSAPSSParamsTemplate DATA ;
+    local:
+       *;
+};
+NSS_3.13 { 	# NSS 3.13 release
+    global:
+SECKEY_RSAPSSParamsTemplate;
 NSS_Get_SECKEY_RSAPSSParamsTemplate;
 NSS_GetVersion;
-;+    local:
-;+       *;
-;+};
-;+NSS_3.13.2 { 	# NSS 3.13.2 release
-;+    global:
+    local:
+       *;
+};
+NSS_3.13.2 { 	# NSS 3.13.2 release
+    global:
 PK11_ImportEncryptedPrivateKeyInfoAndReturnKey;
-;+    local:
-;+       *;
-;+};
-;+NSS_3.14 { 	# NSS 3.14 release
-;+    global:
+    local:
+       *;
+};
+NSS_3.14 { 	# NSS 3.14 release
+    global:
 CERT_CheckOCSPStatus;
 CERT_DecodeOCSPRequest;
 CERT_GetEncodedOCSPResponse;
 PK11_GetBestSlotWithAttributes;
 PK11_GetBestSlotMultipleWithAttributes;
 PK11_PQG_ParamGenV2;
-;+    local:
-;+       *;
-;+};
-;+NSS_3.14.1 {    # NSS 3.14.1 release
-;+    global:
+    local:
+       *;
+};
+NSS_3.14.1 {    # NSS 3.14.1 release
+    global:
 CERT_CreateEncodedOCSPErrorResponse;
 CERT_CreateEncodedOCSPSuccessResponse;
 CERT_CreateOCSPSingleResponseGood;
 CERT_CreateOCSPSingleResponseUnknown;
 CERT_CreateOCSPSingleResponseRevoked;
-;+    local:
-;+       *;
-;+};
-;+NSS_3.14.3 { 	# NSS 3.14.3 release
-;+    global:
+    local:
+       *;
+};
+NSS_3.14.3 { 	# NSS 3.14.3 release
+    global:
 PK11_SignWithSymKey;
-;+    local:
-;+       *;
-;+};
-;+NSS_3.15 { 	# NSS 3.15 release
-;+    global:
+    local:
+       *;
+};
+NSS_3.15 { 	# NSS 3.15 release
+    global:
 CERT_EncodeNameConstraintsExtension;
 PK11_Decrypt;
 PK11_Encrypt;
 CERT_PostOCSPRequest;
 CERT_AddCertToListHead;
-;+    local:
-;+       *;
-;+};
-;+NSS_3.15.4 { 	# NSS 3.15.4 release
-;+    global:
+    local:
+       *;
+};
+NSS_3.15.4 { 	# NSS 3.15.4 release
+    global:
 CERT_ForcePostMethodForOCSP;
 CERT_GetSubjectNameDigest;
 CERT_GetSubjectPublicKeyDigest;
-;+    local:
-;+       *;
-;+};
-;+NSS_3.16.1 { 	# NSS 3.16.1 release
-;+    global:
+    local:
+       *;
+};
+NSS_3.16.1 { 	# NSS 3.16.1 release
+    global:
 PK11_ExportDERPrivateKeyInfo;
 PK11_ExportPrivKeyInfo;
 SECMOD_InternaltoPubMechFlags;
-;+    local:
-;+       *;
-;+};
-;+NSS_3.16.2 { 	# NSS 3.16.2 release
-;+    global:
+    local:
+       *;
+};
+NSS_3.16.2 { 	# NSS 3.16.2 release
+    global:
 CERT_AddExtensionByOID;
 CERT_GetGeneralNameTypeFromString;
 PK11_PubEncrypt;
 PK11_PrivDecrypt;
-;+    local:
-;+       *;
-;+};
+    local:
+       *;
+};
diff --git a/mozilla/security/nss/lib/nss/pkixpriv.def b/mozilla/security/nss/lib/nss/pkixpriv.def
index 440e6d9..815c4c7 100644
--- a/mozilla/security/nss/lib/nss/pkixpriv.def
+++ b/mozilla/security/nss/lib/nss/pkixpriv.def
@@ -1,11 +1,11 @@
-; This Source Code Form is subject to the terms of the Mozilla Public
-; License, v. 2.0. If a copy of the MPL was not distributed with this
-; file, You can obtain one at http://mozilla.org/MPL/2.0/.
+;
+;
+;
 
-;+LIBPKIXprivate {
-;+    global:
-;+# libpkix functions
-;+# May not become part of the NSS public API. Needed for unit testing for now.
+LIBPKIXprivate {
+    global:
+# libpkix functions
+# May not become part of the NSS public API. Needed for unit testing for now.
 NSS_Get_PKIX_PL_LDAPMessageTemplate;
 PKIX_PL_LDAPMessageTemplate;
 PKIX_ALLOC_ERROR;
@@ -95,7 +95,7 @@ PKIX_Error_GetCause;
 PKIX_Error_GetDescription;
 PKIX_Error_GetErrorClass;
 PKIX_Error_GetSupplementaryInfo;
-PKIX_ERRORCLASSNAMES DATA ;
+PKIX_ERRORCLASSNAMES;
 PKIX_Initialize;
 PKIX_Initialize_SetConfigDir;
 PKIX_List_AppendItem;
@@ -319,4 +319,4 @@ PKIX_ResourceLimits_GetMaxTime;
 PKIX_ResourceLimits_SetMaxDepth;
 PKIX_ResourceLimits_SetMaxFanout;
 PKIX_ResourceLimits_SetMaxTime;
-;+};
+};
diff --git a/mozilla/security/nss/lib/smime/smime.def b/mozilla/security/nss/lib/smime/smime.def
index a5e1a37..8ffcd3e 100644
--- a/mozilla/security/nss/lib/smime/smime.def
+++ b/mozilla/security/nss/lib/smime/smime.def
@@ -1,25 +1,21 @@
-;+#
-;+# This Source Code Form is subject to the terms of the Mozilla Public
-;+# License, v. 2.0. If a copy of the MPL was not distributed with this
-;+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
-;+#
-;+# OK, this file is meant to support SUN, LINUX, AIX and WINDOWS
-;+#   1. For all unix platforms, the string ";-"  means "remove this line"
-;+#   2. For all unix platforms, the string " DATA " will be removed from any 
-;+#	line on which it occurs.
-;+#   3. Lines containing ";+" will have ";+" removed on SUN and LINUX.
-;+#      On AIX, lines containing ";+" will be removed.  
-;+#   4. For all unix platforms, the string ";;" will thave the ";;" removed.
-;+#   5. For all unix platforms, after the above processing has taken place,
-;+#    all characters after the first ";" on the line will be removed.  
-;+#    And for AIX, the first ";" will also be removed.
-;+#  This file is passed directly to windows. Since ';' is a comment, all UNIX
-;+#   directives are hidden behind ";", ";+", and ";-"
-;+
-;+NSS_3.2 {               # NSS 3.2 release
-;+    global:
-LIBRARY smime3 ;-
-EXPORTS	;-
+#
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+#
+# OK, this file is meant to support SUN, LINUX, AIX and WINDOWS
+#   2. For all unix platforms, the string "" will be removed from any 
+#	line on which it occurs.
+#   3. Lines containing ";
+#      On AIX, lines containing ";
+#   4. For all unix platforms, the string "" will thave the ";
+#   5. For all unix platforms, after the above processing has taken place,
+#    all characters after the first ";
+#    And for AIX, the first ";
+#  This file is passed directly to windows. Since ';
+
+NSS_3.2 {               # NSS 3.2 release
+    global:
 NSS_CMSContentInfo_GetBulkKey;
 NSS_CMSContentInfo_GetBulkKeySize;
 NSS_CMSContentInfo_GetContent;
@@ -139,25 +135,25 @@ SEC_PKCS12DecoderVerify;
 SEC_PKCS12DestroyExportContext;
 SEC_PKCS12IsEncryptionAllowed;
 SEC_PKCS12SetPreferredCipher;
-;+    local:
-;+        *;
-;+};
-;+NSS_3.2.1 {               # NSS 3.2.1 release
-;+    global:
+    local:
+        *;
+};
+NSS_3.2.1 {               # NSS 3.2.1 release
+    global:
 NSSSMIME_VersionCheck;
-;+    local:
-;+        *;
-;+};
-;+NSS_3.3 {     # NSS 3.3 release
-;+    global:
+    local:
+        *;
+};
+NSS_3.3 {     # NSS 3.3 release
+    global:
 SEC_PKCS7AddCertificate;
 SEC_PKCS7CreateCertsOnly;
 SEC_PKCS7Encode;
-;+    local:
-;+       *;
-;+};
-;+NSS_3.4 {     # NSS 3.4 release
-;+    global:
+    local:
+       *;
+};
+NSS_3.4 {     # NSS 3.4 release
+    global:
 CERT_DecodeCertFromPackage;
 NSS_CMSMessage_IsSigned;
 NSS_CMSSignedData_SetDigestValue;
@@ -173,109 +169,109 @@ SEC_PKCS7GetSigningTime;
 SEC_PKCS7SetContent;
 SEC_PKCS7VerifyDetachedSignature;
 SECMIME_DecryptionAllowed;
-;+    local:
-;+       *;
-;+};
-;+NSS_3.4.1 {     # NSS 3.4.1 release
-;+    global:
+    local:
+       *;
+};
+NSS_3.4.1 {     # NSS 3.4.1 release
+    global:
 NSS_CMSMessage_IsEncrypted;
-;+    local:
-;+       *;
-;+};
-;+NSS_3.6 {     # NSS 3.6 release
-;+    global:
+    local:
+       *;
+};
+NSS_3.6 {     # NSS 3.6 release
+    global:
 NSS_CMSSignerInfo_AddMSSMIMEEncKeyPrefs;
 NSS_CMSSignerInfo_CreateWithSubjKeyID;
-;+    local:
-;+       *;
-;+};
-;+NSS_3.7 {     # NSS 3.7 release
-;+    global:
+    local:
+       *;
+};
+NSS_3.7 {     # NSS 3.7 release
+    global:
 NSS_CMSRecipientInfo_CreateWithSubjKeyID;
 NSS_CMSRecipientInfo_CreateWithSubjKeyIDFromCert;
-;+    local:
-;+       *;
-;+};
-;+NSS_3.7.2 {   # NSS 3.7.2 release
-;+    global:
+    local:
+       *;
+};
+NSS_3.7.2 {   # NSS 3.7.2 release
+    global:
 NSS_CMSRecipientInfo_WrapBulkKey;
 NSS_CMSRecipientInfo_UnwrapBulkKey;
-;+    local:
-;+       *;
-;+};
-;+NSS_3.8 {   # NSS 3.8 release
-;+    global:
+    local:
+       *;
+};
+NSS_3.8 {   # NSS 3.8 release
+    global:
 NSS_CMSRecipientInfo_CreateNew;
 NSS_CMSRecipientInfo_CreateFromDER;
 NSS_CMSRecipientInfo_Encode;
 NSS_CMSRecipientInfo_GetCertAndKey;
 SEC_PKCS12DecoderSetTargetTokenCAs;
-;+    local:
-;+       *;
-;+};
-;+NSS_3.9 {   # NSS 3.9 release
-;+    global:
+    local:
+       *;
+};
+NSS_3.9 {   # NSS 3.9 release
+    global:
 SEC_PKCS7DecoderAbort;
 SEC_PKCS7EncoderAbort;
-;+    local:
-;+       *;
-;+};
-;+NSS_3.9.3 {   # NSS 3.9.3 release
-;+    global:
+    local:
+       *;
+};
+NSS_3.9.3 {   # NSS 3.9.3 release
+    global:
 CERT_ConvertAndDecodeCertificate;
 SEC_PKCS7EncodeItem;
-;+    local:
-;+       *;
-;+};
-;+NSS_3.10 {   # NSS 3.10 release
-;+    global:
+    local:
+       *;
+};
+NSS_3.10 {   # NSS 3.10 release
+    global:
 SEC_PKCS12DecoderIterateInit;
 SEC_PKCS12DecoderIterateNext;
 SEC_PKCS12DecryptionAllowed;
-;+    local:
-;+       *;
-;+};
-;+NSS_3.12.2 {   # NSS 3.12.2 release
-;+    global:
+    local:
+       *;
+};
+NSS_3.12.2 {   # NSS 3.12.2 release
+    global:
 SEC_PKCS12AddCertOrChainAndKey;
-;+    local:
-;+       *;
-;+};
-;+NSS_3.12.10 {   # NSS 3.12.10 release
-;+    global:
+    local:
+       *;
+};
+NSS_3.12.10 {   # NSS 3.12.10 release
+    global:
 NSS_CMSType_RegisterContentType;
 NSS_CMSContentInfo_SetDontStream;
 NSS_SMIMEUtil_CreateMSSMIMEEncKeyPrefs;
-;+#
-;+# Data objects
-;+#
-;+# Don't export these DATA symbols on Windows because they don't work right.
-;+# Use the SEC_ASN1_GET / SEC_ASN1_SUB / SEC_ASN1_XTRN macros to access them.
-;+#
-;+# See nssutil for other examples.
-;+#
-;;NSSCMSGenericWrapperDataTemplate DATA ;
-;;NSS_PointerToCMSGenericWrapperDataTemplate DATA ;
+#
+# Data objects
+#
+# Don't export thesesymbols on Windows because they don't work right.
+# Use the SEC_ASN1_GET / SEC_ASN1_SUB / SEC_ASN1_XTRN macros to access them.
+#
+# See nssutil for other examples.
+#
+NSSCMSGenericWrapperDataTemplate;
+NSS_PointerToCMSGenericWrapperDataTemplate;
 NSS_Get_NSSCMSGenericWrapperDataTemplate;
 NSS_Get_NSS_PointerToCMSGenericWrapperDataTemplate;
-;+    local:
-;+       *;
-;+};
-;+NSS_3.13 {    # NSS 3.13 release
-;+    global:
+    local:
+       *;
+};
+NSS_3.13 {    # NSS 3.13 release
+    global:
 NSSSMIME_GetVersion;
-;+    local:
-;+       *;
-;+};
-;+NSS_3.15 {    # NSS 3.15 release
-;+    global:
+    local:
+       *;
+};
+NSS_3.15 {    # NSS 3.15 release
+    global:
 SEC_PKCS7VerifyDetachedSignatureAtTime;
-;+    local:
-;+       *;
-;+};
-;+NSS_3.16 {    # NSS 3.16 release
-;+    global:
+    local:
+       *;
+};
+NSS_3.16 {    # NSS 3.16 release
+    global:
 NSS_CMSSignerInfo_Verify;
-;+    local:
-;+       *;
-;+};
+    local:
+       *;
+};
diff --git a/mozilla/security/nss/lib/softoken/legacydb/nssdbm.def b/mozilla/security/nss/lib/softoken/legacydb/nssdbm.def
index e4dc6f2..af4fe40 100644
--- a/mozilla/security/nss/lib/softoken/legacydb/nssdbm.def
+++ b/mozilla/security/nss/lib/softoken/legacydb/nssdbm.def
@@ -1,24 +1,20 @@
-;+#
-;+# This Source Code Form is subject to the terms of the Mozilla Public
-;+# License, v. 2.0. If a copy of the MPL was not distributed with this
-;+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
-;+#
-;+# OK, this file is meant to support SUN, LINUX, AIX and WINDOWS
-;+#   1. For all unix platforms, the string ";-"  means "remove this line"
-;+#   2. For all unix platforms, the string " DATA " will be removed from any 
-;+#	line on which it occurs.
-;+#   3. Lines containing ";+" will have ";+" removed on SUN and LINUX.
-;+#      On AIX, lines containing ";+" will be removed.  
-;+#   4. For all unix platforms, the string ";;" will thave the ";;" removed.
-;+#   5. For all unix platforms, after the above processing has taken place,
-;+#    all characters after the first ";" on the line will be removed.  
-;+#    And for AIX, the first ";" will also be removed.
-;+#  This file is passed directly to windows. Since ';' is a comment, all UNIX
-;+#   directives are hidden behind ";", ";+", and ";-"
-;+NSSDBM_3.12 {       # NSS 3.12 release
-;+    global:
-LIBRARY nssdbm3 ;-
-EXPORTS ;-
+#
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+#
+# OK, this file is meant to support SUN, LINUX, AIX and WINDOWS
+#   2. For all unix platforms, the string "" will be removed from any 
+#	line on which it occurs.
+#   3. Lines containing ";
+#      On AIX, lines containing ";
+#   4. For all unix platforms, the string "" will thave the ";
+#   5. For all unix platforms, after the above processing has taken place,
+#    all characters after the first ";
+#    And for AIX, the first ";
+#  This file is passed directly to windows. Since ';
+NSSDBM_3.12 {       # NSS 3.12 release
+    global:
 legacy_Open;
 legacy_Shutdown;
 legacy_ReadSecmodDB;
@@ -26,6 +22,6 @@ legacy_ReleaseSecmodDBData;
 legacy_AddSecmodDB;
 legacy_DeleteSecmodDB;
 legacy_SetCryptFunctions;
-;+    local:
-;+       *;
-;+};
+    local:
+       *;
+};
diff --git a/mozilla/security/nss/lib/softoken/softokn.def b/mozilla/security/nss/lib/softoken/softokn.def
index 24c5915..7cf9926 100644
--- a/mozilla/security/nss/lib/softoken/softokn.def
+++ b/mozilla/security/nss/lib/softoken/softokn.def
@@ -1,28 +1,24 @@
-;+#
-;+# This Source Code Form is subject to the terms of the Mozilla Public
-;+# License, v. 2.0. If a copy of the MPL was not distributed with this
-;+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
-;+#
-;+# OK, this file is meant to support SUN, LINUX, AIX and WINDOWS
-;+#   1. For all unix platforms, the string ";-"  means "remove this line"
-;+#   2. For all unix platforms, the string " DATA " will be removed from any 
-;+#	line on which it occurs.
-;+#   3. Lines containing ";+" will have ";+" removed on SUN and LINUX.
-;+#      On AIX, lines containing ";+" will be removed.  
-;+#   4. For all unix platforms, the string ";;" will thave the ";;" removed.
-;+#   5. For all unix platforms, after the above processing has taken place,
-;+#    all characters after the first ";" on the line will be removed.  
-;+#    And for AIX, the first ";" will also be removed.
-;+#  This file is passed directly to windows. Since ';' is a comment, all UNIX
-;+#   directives are hidden behind ";", ";+", and ";-"
-;+NSS_3.4 {       # NSS 3.4 release
-;+    global:
-LIBRARY softokn3 ;-
-EXPORTS ;-
-C_GetFunctionList; Make this function like a real PKCS #11 module as well
+#
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+#
+# OK, this file is meant to support SUN, LINUX, AIX and WINDOWS
+#   2. For all unix platforms, the string "" will be removed from any 
+#	line on which it occurs.
+#   3. Lines containing ";
+#      On AIX, lines containing ";
+#   4. For all unix platforms, the string "" will thave the ";
+#   5. For all unix platforms, after the above processing has taken place,
+#    all characters after the first ";
+#    And for AIX, the first ";
+#  This file is passed directly to windows. Since ';
+NSS_3.4 {       # NSS 3.4 release
+    global:
+C_GetFunctionList;
 FC_GetFunctionList;
 NSC_GetFunctionList;
 NSC_ModuleDBFunc;
-;+    local:
-;+       *;
-;+};
+    local:
+       *;
+};
diff --git a/mozilla/security/nss/lib/sqlite/sqlite.def b/mozilla/security/nss/lib/sqlite/sqlite.def
index 00fa623..d4dd115 100644
--- a/mozilla/security/nss/lib/sqlite/sqlite.def
+++ b/mozilla/security/nss/lib/sqlite/sqlite.def
@@ -1,24 +1,20 @@
-;+# This Source Code Form is subject to the terms of the Mozilla Public
-;+# License, v. 2.0. If a copy of the MPL was not distributed with this
-;+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
-;+#
-;+#
-;+# OK, this file is meant to support SUN, LINUX, AIX and WINDOWS
-;+#   1. For all unix platforms, the string ";-"  means "remove this line"
-;+#   2. For all unix platforms, the string " DATA " will be removed from any 
-;+#	line on which it occurs.
-;+#   3. Lines containing ";+" will have ";+" removed on SUN and LINUX.
-;+#      On AIX, lines containing ";+" will be removed.  
-;+#   4. For all unix platforms, the string ";;" will thave the ";;" removed.
-;+#   5. For all unix platforms, after the above processing has taken place,
-;+#    all characters after the first ";" on the line will be removed.  
-;+#    And for AIX, the first ";" will also be removed.
-;+#  This file is passed directly to windows. Since ';' is a comment, all UNIX
-;+#   directives are hidden behind ";", ";+", and ";-"
-;+SQLITE_3 {       
-;+    global:
-LIBRARY sqlite3 ;-
-EXPORTS ;-
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+#
+#
+# OK, this file is meant to support SUN, LINUX, AIX and WINDOWS
+#   2. For all unix platforms, the string "" will be removed from any 
+#	line on which it occurs.
+#   3. Lines containing ";
+#      On AIX, lines containing ";
+#   4. For all unix platforms, the string "" will thave the ";
+#   5. For all unix platforms, after the above processing has taken place,
+#    all characters after the first ";
+#    And for AIX, the first ";
+#  This file is passed directly to windows. Since ';
+SQLITE_3 {       
+    global:
 sqlite3_aggregate_context;
 sqlite3_aggregate_count;
 sqlite3_auto_extension;
@@ -120,7 +116,7 @@ sqlite3_set_auxdata;
 sqlite3_sleep;
 sqlite3_snprintf;
 sqlite3_step;
-;;sqlite3_temp_directory DATA ;
+sqlite3_temp_directory;
 sqlite3_thread_cleanup;
 sqlite3_total_changes;
 sqlite3_trace;
@@ -142,6 +138,6 @@ sqlite3_value_type;
 sqlite3_version;
 sqlite3_vmprintf;
 sqlite3_wal_checkpoint;
-;+    local:
-;+       *;
-;+};
+    local:
+       *;
+};
diff --git a/mozilla/security/nss/lib/ssl/ssl.def b/mozilla/security/nss/lib/ssl/ssl.def
index 3a2340b..870853b 100644
--- a/mozilla/security/nss/lib/ssl/ssl.def
+++ b/mozilla/security/nss/lib/ssl/ssl.def
@@ -1,27 +1,23 @@
-;+#
-;+# This Source Code Form is subject to the terms of the Mozilla Public
-;+# License, v. 2.0. If a copy of the MPL was not distributed with this
-;+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
-;+#
-;+# OK, this file is meant to support SUN, LINUX, AIX and WINDOWS
-;+#   1. For all unix platforms, the string ";-"  means "remove this line"
-;+#   2. For all unix platforms, the string " DATA " will be removed from any 
-;+#     line on which it occurs.
-;+#   3. Lines containing ";+" will have ";+" removed on SUN and LINUX.
-;+#      On AIX, lines containing ";+" will be removed.
-;+#   4. For all unix platforms, the string ";;" will thave the ";;" removed.
-;+#   5. For all unix platforms, after the above processing has taken place,
-;+#    all characters after the first ";" on the line will be removed.
-;+#    And for AIX, the first ";" will also be removed.
-;+#  This file is passed directly to windows. Since ';' is a comment, all UNIX
-;+#   directives are hidden behind ";", ";+", and ";-"
-;+
-;+NSS_3.2 {       # NSS 3.2 release
-;+    global:
-LIBRARY ssl3 ;-
-EXPORTS ;-
-SSL_ImplementedCiphers DATA ;
-SSL_NumImplementedCiphers DATA ;
+#
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+#
+# OK, this file is meant to support SUN, LINUX, AIX and WINDOWS
+#   2. For all unix platforms, the string "" will be removed from any 
+#     line on which it occurs.
+#   3. Lines containing ";
+#      On AIX, lines containing ";
+#   4. For all unix platforms, the string "" will thave the ";
+#   5. For all unix platforms, after the above processing has taken place,
+#    all characters after the first ";
+#    And for AIX, the first ";
+#  This file is passed directly to windows. Since ';
+
+NSS_3.2 {       # NSS 3.2 release
+    global:
+SSL_ImplementedCiphers;
+SSL_NumImplementedCiphers;
 NSS_CmpCertChainWCANames;
 NSS_FindCertKEAType;
 NSS_GetClientAuthData;
@@ -69,46 +65,46 @@ SSL_SecurityStatus;
 SSL_SetPKCS11PinArg;
 SSL_SetSockPeerID;
 SSL_SetURL;
-;+    local:
-;+*;
-;+};
-;+NSS_3.2.1 {       # NSS 3.2.1 release
-;+    global:
+    local:
+*;
+};
+NSS_3.2.1 {       # NSS 3.2.1 release
+    global:
 NSSSSL_VersionCheck;
-;+    local:
-;+*;
-;+};
-;+NSS_3.4 {         # NSS 3.4   release
-;+    global:
+    local:
+*;
+};
+NSS_3.4 {         # NSS 3.4   release
+    global:
 SSL_GetChannelInfo;
 SSL_GetCipherSuiteInfo;
 SSL_GetMaxServerCacheLocks;
 SSL_LocalCertificate;
 SSL_SetMaxServerCacheLocks;
-;+    local:
-;+*;
-;+};
-;+NSS_3.7.4 {       # NSS 3.7.4 release
-;+    global:
+    local:
+*;
+};
+NSS_3.7.4 {       # NSS 3.7.4 release
+    global:
 SSL_ShutdownServerSessionIDCache;
-;+    local:
-;+*;
-;+};
-;+NSS_3.11.4 {      # NSS 3.11.4 release
-;+    global:
+    local:
+*;
+};
+NSS_3.11.4 {      # NSS 3.11.4 release
+    global:
 SSL_ForceHandshakeWithTimeout;
 SSL_ReHandshakeWithTimeout;
-;+    local:
-;+*;
-;+};
-;+NSS_3.11.8 {      # NSS 3.11.8 release
-;+    global:
+    local:
+*;
+};
+NSS_3.11.8 {      # NSS 3.11.8 release
+    global:
 SSL_CanBypass;
-;+    local:
-;+*;
-;+};
-;+NSS_3.12.6 {      # NSS 3.12.6 release
-;+    global:
+    local:
+*;
+};
+NSS_3.12.6 {      # NSS 3.12.6 release
+    global:
 SSL_ConfigServerSessionIDCacheWithOpt;
 SSL_GetImplementedCiphers;
 SSL_GetNegotiatedHostInfo;
@@ -117,32 +113,32 @@ SSL_HandshakeNegotiatedExtension;
 SSL_ReconfigFD;
 SSL_SetTrustAnchors;
 SSL_SNISocketConfigHook;
-;+    local:
-;+*;
-;+};
-;+NSS_3.12.10 {      # NSS 3.12.10 release
-;+    global:
+    local:
+*;
+};
+NSS_3.12.10 {      # NSS 3.12.10 release
+    global:
 SSL_ConfigSecureServerWithCertChain;
-;+    local:
-;+*;
-;+};
-;+NSS_3.13 {    # NSS 3.13 release
-;+    global:
+    local:
+*;
+};
+NSS_3.13 {    # NSS 3.13 release
+    global:
 NSSSSL_GetVersion;
-;+    local:
-;+       *;
-;+};
-;+NSS_3.13.2 {    # NSS 3.13.2 release
-;+    global:
+    local:
+       *;
+};
+NSS_3.13.2 {    # NSS 3.13.2 release
+    global:
 SSL_SetNextProtoCallback;
 SSL_SetNextProtoNego;
 SSL_GetNextProto;
 SSL_AuthCertificateComplete;
-;+    local:
-;+       *;
-;+};
-;+NSS_3.14 {      # NSS 3.14 release
-;+    global:
+    local:
+       *;
+};
+NSS_3.14 {      # NSS 3.14 release
+    global:
 DTLS_GetHandshakeTimeout;
 DTLS_ImportFD;
 SSL_ExportKeyingMaterial;
@@ -153,21 +149,21 @@ SSL_VersionRangeSet;
 SSL_VersionRangeSetDefault;
 SSL_GetSRTPCipher;
 SSL_SetSRTPCiphers;
-;+    local:
-;+*;
-;+};
-;+NSS_3.15 {      # NSS 3.15 release
-;+    global:
+    local:
+*;
+};
+NSS_3.15 {      # NSS 3.15 release
+    global:
 SSL_PeerStapledOCSPResponses;
 SSL_SetStapledOCSPResponses;
-;+    local:
-;+*;
-;+};
-;+NSS_3.15.4 {    # NSS 3.15.4 release
-;+    global:
+    local:
+*;
+};
+NSS_3.15.4 {    # NSS 3.15.4 release
+    global:
 SSL_PeerCertificateChain;
 SSL_RecommendedCanFalseStart;
 SSL_SetCanFalseStartCallback;
-;+    local:
-;+*;
-;+};
+    local:
+*;
+};
diff --git a/mozilla/security/nss/lib/util/nssutil.def b/mozilla/security/nss/lib/util/nssutil.def
index 86a0ad7..16d30d8 100644
--- a/mozilla/security/nss/lib/util/nssutil.def
+++ b/mozilla/security/nss/lib/util/nssutil.def
@@ -1,24 +1,20 @@
-;+#
-;+# This Source Code Form is subject to the terms of the Mozilla Public
-;+# License, v. 2.0. If a copy of the MPL was not distributed with this
-;+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
-;+#
-;+# OK, this file is meant to support SUN, LINUX, AIX and WINDOWS
-;+#   1. For all unix platforms, the string ";-"  means "remove this line"
-;+#   2. For all unix platforms, the string " DATA " will be removed from any 
-;+#	line on which it occurs.
-;+#   3. Lines containing ";+" will have ";+" removed on SUN and LINUX.
-;+#      On AIX, lines containing ";+" will be removed.  
-;+#   4. For all unix platforms, the string ";;" will thave the ";;" removed.
-;+#   5. For all unix platforms, after the above processing has taken place,
-;+#    all characters after the first ";" on the line will be removed.  
-;+#    And for AIX, the first ";" will also be removed.
-;+#  This file is passed directly to windows. Since ';' is a comment, all UNIX
-;+#   directives are hidden behind ";", ";+", and ";-"
-;+NSSUTIL_3.12 {       # NSS Utilities 3.12 release
-;+    global:
-LIBRARY nssutil3	;-
-EXPORTS		;-
+#
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+#
+# OK, this file is meant to support SUN, LINUX, AIX and WINDOWS
+#   2. For all unix platforms, the string "" will be removed from any 
+#	line on which it occurs.
+#   3. Lines containing ";
+#      On AIX, lines containing ";
+#   4. For all unix platforms, the string "" will thave the ";
+#   5. For all unix platforms, after the above processing has taken place,
+#    all characters after the first ";
+#    And for AIX, the first ";
+#  This file is passed directly to windows. Since ';
+NSSUTIL_3.12 {       # NSS Utilities 3.12 release
+    global:
 ATOB_AsciiToData_Util;
 ATOB_ConvertAsciiToItem_Util;
 BTOA_ConvertItemToAscii_Util;
@@ -142,36 +138,36 @@ SGN_CopyDigestInfo_Util;
 SGN_CreateDigestInfo_Util;
 SGN_DecodeDigestInfo;
 SGN_DestroyDigestInfo_Util;
-;+#
-;+# Data objects
-;+#
-;+# Don't export these DATA symbols on Windows because they don't work right.
-;+# Use the SEC_ASN1_GET / SEC_ASN1_SUB / SEC_ASN1_XTRN macros to access them.
-;;SEC_AnyTemplate_Util DATA ;
-;;SEC_BitStringTemplate_Util DATA ;
-;;SEC_BMPStringTemplate_Util DATA ;
-;;SEC_BooleanTemplate_Util DATA ;
-;;SEC_EnumeratedTemplate DATA ;
-;;SEC_GeneralizedTimeTemplate_Util DATA ;
-;;SEC_IA5StringTemplate_Util DATA ;
-;;SEC_IntegerTemplate_Util DATA ;
-;;SEC_NullTemplate_Util DATA ;
-;;SEC_ObjectIDTemplate_Util DATA ;
-;;SEC_OctetStringTemplate_Util DATA ;
-;;SEC_PointerToAnyTemplate_Util DATA ;
-;;SEC_PointerToEnumeratedTemplate DATA ;
-;;SEC_PointerToGeneralizedTimeTemplate DATA ;
-;;SEC_PointerToOctetStringTemplate_Util DATA ;
-;;SEC_PrintableStringTemplate DATA ;
-;;SEC_SequenceOfAnyTemplate DATA ;
-;;SEC_SequenceOfObjectIDTemplate DATA ;
-;;SEC_SetOfAnyTemplate_Util DATA ;
-;;SEC_SkipTemplate DATA ;
-;;SEC_T61StringTemplate DATA ;
-;;SEC_UniversalStringTemplate DATA ;
-;;SEC_UTF8StringTemplate_Util DATA ;
-;;SECOID_AlgorithmIDTemplate_Util DATA ;
-;;sgn_DigestInfoTemplate_Util DATA ;
+#
+# Data objects
+#
+# Don't export thesesymbols on Windows because they don't work right.
+# Use the SEC_ASN1_GET / SEC_ASN1_SUB / SEC_ASN1_XTRN macros to access them.
+SEC_AnyTemplate_Util;
+SEC_BitStringTemplate_Util;
+SEC_BMPStringTemplate_Util;
+SEC_BooleanTemplate_Util;
+SEC_EnumeratedTemplate;
+SEC_GeneralizedTimeTemplate_Util;
+SEC_IA5StringTemplate_Util;
+SEC_IntegerTemplate_Util;
+SEC_NullTemplate_Util;
+SEC_ObjectIDTemplate_Util;
+SEC_OctetStringTemplate_Util;
+SEC_PointerToAnyTemplate_Util;
+SEC_PointerToEnumeratedTemplate;
+SEC_PointerToGeneralizedTimeTemplate;
+SEC_PointerToOctetStringTemplate_Util;
+SEC_PrintableStringTemplate;
+SEC_SequenceOfAnyTemplate;
+SEC_SequenceOfObjectIDTemplate;
+SEC_SetOfAnyTemplate_Util;
+SEC_SkipTemplate;
+SEC_T61StringTemplate;
+SEC_UniversalStringTemplate;
+SEC_UTF8StringTemplate_Util;
+SECOID_AlgorithmIDTemplate_Util;
+sgn_DigestInfoTemplate_Util;
 NSS_Get_SEC_AnyTemplate_Util;
 NSS_Get_SEC_BitStringTemplate_Util;
 NSS_Get_SEC_BMPStringTemplate_Util;
@@ -197,44 +193,44 @@ NSS_Get_SEC_UniversalStringTemplate;
 NSS_Get_SEC_UTF8StringTemplate_Util;
 NSS_Get_SECOID_AlgorithmIDTemplate_Util;
 NSS_Get_sgn_DigestInfoTemplate_Util;
-;+    local:
-;+       *;
-;+};
-;+NSSUTIL_3.12.3 {       # NSS Utilities 3.12.3 release
-;+    global:
+    local:
+       *;
+};
+NSSUTIL_3.12.3 {       # NSS Utilities 3.12.3 release
+    global:
 NSS_GetAlgorithmPolicy;
 NSS_SetAlgorithmPolicy;
 SECITEM_ReallocItem;
 UTIL_SetForkState;
-;+    local:
-;+       *;
-;+};
-;+NSSUTIL_3.12.5 {       # NSS Utilities 3.12.5 release
-;+    global:
+    local:
+       *;
+};
+NSSUTIL_3.12.5 {       # NSS Utilities 3.12.5 release
+    global:
 NSS_SecureMemcmp;
 PORT_LoadLibraryFromOrigin;
-;+    local:
-;+       *;
-;+};
-;+NSSUTIL_3.12.7 {       # NSS Utilities 3.12.7 release
-;+    global:
+    local:
+       *;
+};
+NSSUTIL_3.12.7 {       # NSS Utilities 3.12.7 release
+    global:
 PORT_RegExpSearch;
-;+    local:
-;+       *;
-;+};
-;+NSSUTIL_3.13 {         # NSS Utilities 3.13 release
-;+    global:
+    local:
+       *;
+};
+NSSUTIL_3.13 {         # NSS Utilities 3.13 release
+    global:
 NSSUTIL_GetVersion;
 NSS_InitializePRErrorTable;
-;+    local:
-;+       *;
-;+};
-;+NSSUTIL_3.14 {         # NSS Utilities 3.14 release
-;+    global:
-;+# private exports for softoken
+    local:
+       *;
+};
+NSSUTIL_3.14 {         # NSS Utilities 3.14 release
+    global:
+# private exports for softoken
 _NSSUTIL_GetSecmodName;
 _NSSUTIL_EvaluateConfigDir;
-;+# public exports
+# public exports
 NSSUTIL_ArgDecodeNumber;
 NSSUTIL_ArgFetchValue;
 NSSUTIL_ArgGetParamValue;
@@ -258,16 +254,16 @@ NSSUTIL_MkNSSString;
 NSSUTIL_MkSlotString;
 NSSUTIL_Quote;
 NSSUTIL_QuoteSize;
-;+    local:
-;+       *;
-;+};
-;+NSSUTIL_3.15 {         # NSS Utilities 3.15 release
-;+    global:
+    local:
+       *;
+};
+NSSUTIL_3.15 {         # NSS Utilities 3.15 release
+    global:
 SECITEM_AllocArray;
 SECITEM_DupArray;
 SECITEM_FreeArray;
 SECITEM_ReallocItemV2;
 SECITEM_ZfreeArray;
-;+    local:
-;+       *;
-;+};
+    local:
+       *;
+};
